### ISSUE TITLE
Include Mandrill

### ISSUE THREAD

**Issue by stefanisak** [gianlucadelgobbo/avnode#39]

**Comment by gianlucadelgobbo** [comment]
Do we have to pay for it?


Gianluca
______________________________________
Flyer communication
www.flyer.it
Via del Verano 39 - 00185 Roma
Tel. +39 06 78147301 Fax +39 06 78390805

**Comment by stefanisak** [comment]
I thought Mandrill is the way to go, as I saw that you are using Mailchimp already. You find everything about pricing here http://mandrill.com/pricing/

If you want everything for free, maybe https://www.mailgun.com/ is an option.

Please elaborate on that...

**Comment by gianlucadelgobbo** [comment]
using gmail is crazy?

we need to send only service emails, registrations, lost passwords and
notifications, is too much for gmail, we have a business account


Gianluca
______________________________________
Flyer communication
www.flyer.it
Via del Verano 39 - 00185 Roma
Tel. +39 06 78147301 Fax +39 06 78390805

**Comment by stefanisak** [comment]
I wouldn't say crazy, but to me it doesn't feel right. Can you create separate access keys? Otherwise you need to be aware, that your email account credentials will be stored on the server. And it also doesn't solve the problem, that you need emails while developing.

**Comment by gianlucadelgobbo** [comment]
ok lets go mandrill


Gianluca
______________________________________
Flyer communication
www.flyer.it
Via del Verano 39 - 00185 Roma
Tel. +39 06 78147301 Fax +39 06 78390805

**Comment by gianlucadelgobbo** [comment]
wait we spend 300 euro on mailchimp I was thinking to use amazon there is
something similar in amazon ses?


Gianluca
______________________________________
Flyer communication
www.flyer.it
Via del Verano 39 - 00185 Roma
Tel. +39 06 78147301 Fax +39 06 78390805

**Comment by alappe** [comment]
Yes, amazon ses should be similar and do the same… I haven't used it yet, but this would be a way.

**Comment by gianlucadelgobbo** [comment]
So lets try amazon ses?
How is possible to send newsletter with it?

**Comment by stefanisak** [comment]
@gianlucadelgobbo FYI SES is still running in sandbox mode. This causes that emails can only be send to verified email address:

```
Your Amazon SES account has "sandbox" access in region EU (Ireland). With sandbox access you can only send email to the Amazon SES mailbox simulator and to email addresses or domains that you have verified. To be moved out of the sandbox, please request a sending limit increase. Learn more.
Can't find your existing account settings? Your account may be set up in a different AWS region. Try switching regions in the upper right corner of the console.
```

**Comment by gianlucadelgobbo** [comment]
request done

**Comment by stefanisak** [comment]
It's waiting for amazon processing the request.
